MyArrayList.java

```diff
@@ -1 +1,110 @@
+package com.company;
 
+public class MyArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private int position = 0;
+    private Object[] array = new Object[DEFAULT_CAPACITY];
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int size = array.length;
+
+    @Override
+    public void add(T value) {
+        checkForAdd(position);
+        add(value, position);
+    }
+
+    private void checkForAdd(int index) {
+        if (index < 0) {
+            throw new NoSuchFieldError();
```

Comment:
doesn't very good exception for this purpose

Reply:
I changed it 
= new NoSuchElementException("The index is wrong.")
Am I right?